client/src/components/widgets/ContextMenu/script.js

```diff
@@ -142,4 +142,17 @@ export default {
       }
     },
   },
+
+  created() {
+    this.mathJaxOptions = {
+      tex2jax: {
+        inlineMath: [
+          ["$", "$"],
+          ["(", ")"],
+        ],
+        processEscapes: true,
+        processEnvironments: true,
+      },
+    };
+  },
```

Comment:
Let have a single copy of this rather than copying it.

Reply:
Moved into the UI state now

Reply:
This was factored out into the UI store to be re-usable across components.